### ISSUE TITLE
Search Filter : ERROR TypeError: Cannot read property 'toString' of null

### ISSUE THREAD

**Issue by FirassKoubaa** [CuppaLabs/angular2-multiselect-dropdown#228]
the search filter is no longer working 

My scenarios is the following : i m loading options from a WS , the initial loading , selection , deselection are all working , but the search filtering by keyword is throwing this error : 

```
ERROR TypeError: Cannot read property 'toString' of null
    at ListFilterPipe.applyFilter (list-filter.js:16)
    at eval (list-filter.js:10)
    at Array.filter (<anonymous>)
    at ListFilterPipe.transform (list-filter.js:10)
    at Object.eval [as updateDirectives] (AngularMultiSelect.html:54)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:14655)
    at checkAndUpdateView (core.js:13802)
    at callViewAction (core.js:14153)
    at execEmbeddedViewsAction (core.js:14111)
    at checkAndUpdateView (core.js:13803)
```

**Comment by CuppaLabs** [comment]
Fix in pipeline for this week release.

**Comment by CuppaLabs** [comment]
Fixed. available in v2.9.0. New search algorithm
